### PR TITLE
Moving countdown content to CONTRIBUTING.md.

### PR DESCRIPTION
**Pull request by samrobbins**
Cleanup the countdown docs and integrate them with our CONTRIBUTING.md doc.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,6 +88,172 @@ One of the best ways we can keep fastlane an approachable, stable, and dependabl
 
 Writing a custom action is an easy way to extend the capabilities of fastlane. Actions that make good candidates for inclusion in the fastlane codebase are **flexible** and apply to **many projects, teams, and development setups**. Before working to contribute your custom action to fastlane, consider whether it is likely to solve a problem that many developers have. If not, it can still provide value for your fastlane environment! Check out the documentation for creating [local action extensions](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/README.md#extensions).
 
+## Getting started
```
Comment (KrauseFx) [inline CONTRIBUTING.md]: I wonder if we might want to put this up a little bit? I wouldn't want to read all those docs before I can actually see how I can contribute. cc @mfurtak

Comment (samrobbins) [inline CONTRIBUTING.md]: @KrauseFx Yes I think that is a valid concern. Should this be two top level documents?

Comment (mfurtak) [inline CONTRIBUTING.md]: Any restructuring folks want to do is :+1: by me!

Comment (KrauseFx) [inline CONTRIBUTING.md]: Perfect, thanks :+1:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
+
+```
+rake pull
+```
+
+## Submit a pull request
+
+To submit the changes to the fastlane main repo, you have to do the following:
+
+- Squash your commits into one. For example,  to squash three commits into one, do the following: `$ git rebase -i HEAD~3`. In the text editor that comes up, replace the words "pick" with "squash" next to the commits you want to squash. Save and close the editor. For more information, take a look at [7.6 Git Tools - Rewriting History](http://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)
```
Comment (ohwutup) [inline CONTRIBUTING.md]: looks like you have an extra space in between `For example,` and `to squash...`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -88,6 +107,136 @@ One of the best ways we can keep fastlane an approachable, stable, and dependabl
 
 Writing a custom action is an easy way to extend the capabilities of fastlane. Actions that make good candidates for inclusion in the fastlane codebase are **flexible** and apply to **many projects, teams, and development setups**. Before working to contribute your custom action to fastlane, consider whether it is likely to solve a problem that many developers have. If not, it can still provide value for your fastlane environment! Check out the documentation for creating [local action extensions](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/README.md#extensions).
 
+## Developing
+
+When working on something, directly edit the Ruby files in the project folders. Make sure to switch to 2 spaces in your text editor.
```
Comment (ohwutup) [inline CONTRIBUTING.md]: What does this mean to `switch to 2 spaces in your text editor`? Is this like "tabs are spaces"?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -88,6 +107,136 @@ One of the best ways we can keep fastlane an approachable, stable, and dependabl
...
+
+and then run these tests only using
+
+```sh
+rspec -t now
+```
+
+## Running the local code
+
+Run your local copy using
```
Comment (ohwutup) [inline CONTRIBUTING.md]: Looks like this is duplicated from above in the `## Developing` section

File: CONTRIBUTING.md (hunk 3 above)
Comment (samrobbins) [inline CONTRIBUTING.md]: Yeah that's not super clear. I'll update. I don't want to spark the age old debate but we are using spaces right @KrauseFx?

**Comment by ohwutup** [comment]
:rocket: :guitar:

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,13 +1,32 @@
...
+- On your terminal, navigate to the project and run `git remote add upstream https://github.com/fastlane/fastlane` (or use the `git` URL if you use private key auth)
+- Create your feature branch (`git checkout -b my-new-feature`).
+- Commit your changes (`git commit -am 'Add some feature'`).
+- Push to the branch (`git push origin my-new-feature`).
+- Create a new Pull Request.
+
+Before working on something, make sure to have pulled the latest changes. To pull the changes run
+
+```
+rake pull
```
Comment (KrauseFx) [inline CONTRIBUTING.md]: That's not relevant for mono repo any more, right?

Comment (KrauseFx) [inline CONTRIBUTING.md]: Should be `git pull master`?

**Comment by samrobbins** [comment]
@KrauseFx ready for another look. Thx!

**Comment by KrauseFx** [comment]
Looks good to me :+1:

**Pull request merged**